pkgs/by-name/lm/lmstudio/package.nix

```diff
@@ -36,7 +36,7 @@ if stdenv.hostPlatform.isDarwin then
     url =
       args.url
         or "https://installers.lmstudio.ai/darwin/arm64/${version}-${rev}/LM-Studio-${version}-${rev}-arm64.dmg";
-    hash = args.hash or "sha256-kXH3tAazEtl019IBxuavEI9QUamEH3b6UFYRYAO3Fxs=";
+    hash = args.hash or "sha256-say/EXhbjnfClD2JvLRxavFhCqw6IebGPP0iNd5v9pk=";
```

Comment:
Ah, is this right?

Reply:
This is what I used, is it wrong?
```sh
nix hash to-base64 --type sha256 "sha256:$(nix-prefetch-url "https://installers.lmstudio.ai/darwin/arm64/0.3.12-1/LM-Studio-0.3.12-1-arm64.dmg")"
```

Reply:
I got the same thing, the forward slash just threw me off. Carry on! :)